### PR TITLE
Aact 150 outcome analyses groups

### PR DESCRIPTION
**Pull request by tibbs001**

### CODE REVIEW

File: app/models/study.rb [hunk 1]
```diff
@@ -10,6 +10,11 @@ def self.current_interventional
     self.interventional and self.current
   end
 
+  def self.try(nct_id='NCT00023673')
```
Comment (garrettqmartin8) [inline app/models/study.rb]: `try` is a method in the `ActiveSupport` module, so this method will overwrite it. What is the purpose of this method?

File: app/models/study.rb [hunk 2]
```diff
@@ -285,6 +289,30 @@ def get_date(str)
...
+
+  def drop_withdrawals
+    DropWithdrawal.where('nct_id=?',nct_id)
+  end
+
+  def reported_events
+    ReportedEvent.where('nct_id=?',nct_id)
+  end
+
+  def outcomes
```
Comment (garrettqmartin8) [inline app/models/study.rb]: These methods are already defined when we declare `has_many :outcomes`

**Comment by garrettqmartin8** [comment]
👍 once conflicts are resolved.

File: app/models/study.rb (hunk 1 above)
Comment (tibbs001) [inline app/models/study.rb]: The method is a quick way to create the study via a console session.  Just for testing.  I'll remove it.

**Pull request merged**